doc/source/api/genotypephenotype.rst

```diff
@@ -0,0 +1,528 @@
+Summary
+=======
+
+This API endpoints allows users to search for genotype-phenotype
+associations in a GA4GH datastore. The user can search for associations
+by building queries composed of features, phenotypes, and/or evidence
+terms. The API is designed to accommodate search terms specified as
+either a string, external identifier, ontology identifier, or as an
```

Comment:
Is this still the case? The query patterns seem to have drifted from the docs here.